sdk/dnsresolver/Azure.ResourceManager.DnsResolver/Azure.ResourceManager.DnsResolver.sln

```diff
@@ -1,21 +1,17 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.30309.148
+# Visual Studio Version 17
+VisualStudioVersion = 17.3.32901.215
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{6DDC611C-0DBD-497F-8854-19B000044FC3}") = "Azure.ResourceManager.DnsResolver", "src\Azure.ResourceManager.DnsResolver.csproj", "{6D602E02-6AAD-4B8A-939C-F04797160431}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.DnsResolver", "src\Azure.ResourceManager.DnsResolver.csproj", "{6D602E02-6AAD-4B8A-939C-F04797160431}"
 EndProject
-Project("{6DDC611C-0DBD-497F-8854-19B000044FC3}") = "Azure.ResourceManager.DnsResolver.Tests", "tests\Azure.ResourceManager.DnsResolver.Tests.csproj", "{B674F8DC-C1E8-41D7-9183-A53C25AAB958}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.ResourceManager.DnsResolver.Tests", "tests\Azure.ResourceManager.DnsResolver.Tests.csproj", "{B674F8DC-C1E8-41D7-9183-A53C25AAB958}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Azure.Core.TestFramework", "..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{CFD82F5B-3E15-48A1-9125-1C1E5A92573D}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.Core.TestFramework", "..\..\core\Azure.Core.TestFramework\src\Azure.Core.TestFramework.csproj", "{CFD82F5B-3E15-48A1-9125-1C1E5A92573D}"
```

Comment:
I don't think we need `Azure.Core.TestFramework` project in this sln, we have added it for all the RPs. If you meet build error in vs that complain it can't find the test framework, please run `dotnet build` in the terminal first.